lib/apollo-federation/has_directives.rb

```diff
@@ -9,7 +9,7 @@ def add_directive(name:, arguments: nil)
       @federation_directives << { name: name, arguments: arguments }
     end
 
-    def to_graphql
+    def to_graphql(*args, **kwargs)
```

Comment:
Self note: Check out the definition change in `#to_graphql` and how this method is used